hardware_interface/source/skytraq_binary.c

```diff
@@ -307,13 +310,13 @@ GPS_RETURNSTATE skytraq_query_software_version() {
     return skytraq_send_message(payload, length);
 }
 
-GPS_RETURNSTATE skytraq_query_software_CRC(uint8_t *reply) {
+GPS_RETURNSTATE skytraq_query_software_CRC(uint8_t *reply, uint16_t reply_len) {
```

Comment:
I still feel like these (this and the gps time one below) should do parsing of the reply to expose the value that the function name suggests it would, in this case the CRC, instead of pushing all of the reply up to other layers of software to deal with.

Reply:
This follows "query software crc" as in the documentation from skytraq. It creates a packet to query CRC and sends it then updates the reply array. The abstraction from that is the other function you commented on